src/schema/TypeORMConverter.ts

```diff
@@ -265,7 +278,10 @@ export function entityToWhereInput(entity: EntityMetadata): string {
         @TypeGraphQLField({ nullable: true })
         ${column.propertyName}_lte?: ${tsType};
       `;
-    } else {
+    } else if (graphQLDataType !== 'JSONObject') {
+      // @@@ dcaddigan not sure what it means to search by JSONObjects
+      // future release?
```

Comment:
This is interesting.  The way you search into JSON blobs is with a dot operator (at least in Postgres).  If you don't need to search within JSON fields I'd say let's hold off for this PR and we can add later?

Reply:
I'd like to hold off so I can unblock some other things I've been working on.